libvmi_extension/exit_handler/exit_handler.cpp

```diff
@@ -29,12 +29,12 @@ typedef enum hstatus {
     HSTATUS_FAILURE
 } hstatus_t;
 
-#define HCALL_INVALID       0
-#define HCALL_ACK           1
-#define HCALL_GET_REGISTERS 2
-#define HCALL_SET_REGISTERS 3
-#define HCALL_TRANSLATE_V2P 4
-#define HCALL_MAP_PA        5
+#define HCALL_INVALID		0xbf05000000000000
+#define HCALL_ACK		0xbf05000000000001
+#define HCALL_GET_REGISTERS	0xbf05000000000002
+#define HCALL_SET_REGISTERS	0xbf05000000000003
+#define HCALL_TRANSLATE_V2P	0xbf05000000000004
+#define HCALL_MAP_PA		0xbf05000000000005
```

Comment:
Why does it need to be prefixed now with `bf0500000000000`?

Reply:
since we want to support boxy and boxy has their own opcode for each vmcall (for example: https://github.com/Bareflank/boxy/blob/master/bfsdk/include/bfhypercall.h#L65), last time I asked Rian, he said I can use 5 for VMI related stuffs. But maybe I need to confirm with him again.

Reply:
OK, that makes sense